### PR TITLE
BN position feature addition

### PR DESCRIPTION
**Pull request by 3Dgoo**
This pull request adds config control for the Better Navigator position and translucency as described in issue #51

### CODE REVIEW

**Comment by 3Dgoo** [comment]
Hi @jonom,

This change contains the CSS changes from the previous closed PR. I merged those into my branch and decided I would keep them or myself. That was the base for these changes.

Let me know if you want me to undo those CSS changes from the PR and I'll take care of that.

**Comment by jonom** [comment]
Hi @3Dgoo, yes if you don't mind undoing all of the ID->Class linter changes (and var -> let/const), that would be great.

**Comment by 3Dgoo** [comment]
Hi @jonom. Sure thing. I have removed undone the requested changes and just left the new feature. Have a look and let me know what you think.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,20 @@ You can mark certain CMS users as developers in your site's config, so they can
 
 ## Customisation
 
+BetterNavigator can be made translucent by adding the following config setting:
```
Comment (jonom) [inline README.md:60]: Edit: can be made translucent **when collapsed** by adding

**Review by jonom — CHANGES_REQUESTED** [review]
Thanks @3Dgoo looking good. Just some minor tweaks to make.

File: src/Extension/BetterNavigatorExtension.php [hunk 2]
```diff
@@ -180,4 +181,27 @@ private function isAPage()
             && $this->owner->dataRecord instanceof SiteTree
             && $this->owner->dataRecord->ID > 0;
     }
+
+
+    /**
+     * @return string
+     */
+    private function cssClass() {
```
[the remark this answers is not in the document]
Comment (3Dgoo) [reply src/Extension/BetterNavigatorExtension.php]: Sure thing. This is done.

File: src/Extension/BetterNavigatorExtension.php [hunk 3]
```diff
@@ -131,6 +131,7 @@ private function generateNavigator()
             'Member' => $member,
             'Stage' => Versioned::get_stage(),
             'Viewing' => $viewing, // What we're viewing doesn't necessarily align with the active Stage
+            'CssClass' => $this->cssClass(),
```
[the remark this answers is not in the document]
Comment (3Dgoo) [reply src/Extension/BetterNavigatorExtension.php]: Sure thing. This is done.

File: README.md (hunk 1 above)
Comment (3Dgoo) [reply README.md]: Sure thing. This is done.

**Review by jonom — APPROVED** [review]
Thanks @3Dgoo, I just tweaked the docs a little as the example positions were off. Works well!

**Comment by jonom** [comment]
Tagged 5.2.0. Thanks again.

**Comment by 3Dgoo** [comment]
Thanks @jonom

**Pull request merged**